src/buffer/out/AttrRow.cpp

```diff
@@ -250,11 +250,11 @@ void ATTR_ROW::ReplaceAttrs(const TextAttribute& toBeReplacedAttr, const TextAtt
     if (newAttrs.size() == 1)
     {
         // Get the new color attribute we're trying to apply
-        const TextAttribute NewAttr = newAttrs.at(0).GetAttributes();
+        const TextAttribute NewAttr = newAttrs[0].GetAttributes();

```

Comment:
when this switches to gsl::span it's 100% going to be index-checked again. is this really meaningful for perf? if so, we need to revisit #6921.

There's no way to use gsl::span _unchecked._ Even with `til::at`.